v19.2/import-into.md

```diff
@@ -184,23 +182,18 @@ Google Cloud:
 ## Known limitations
 
 - Only CSV data importing into a single table is supported.
+- During incremental import, the table being imported into is taken offline.
+- After incremental import, [constraints](constraints.html) will be un-validated and need to be [re-validated](validate-constraint.html).
 - `IMPORT INTO` **cannot**:
     - Be used within a [transaction](transactions.html).
     - Replace a row.
     - Add a row that conflicts with a `UNIQUE` [index](indexes.html).
-- Currently, `IMPORT INTO` does not work on indexed tables.
-- `IMPORT` can sometimes fail with a "context canceled" error, or can restart itself many times without ever finishing. If this is happening, it is likely due to a high amount of disk contention. This can be mitigated by setting the `kv.bulk_io_write.max_rate` [cluster setting](cluster-settings.html) to a value below your max disk write speed. For example, to set it to 10MB/s, execute:
+    - Be used on [indexed](indexes.html) or [interleaved](interleave-in-parent.html) tables.
```

Comment:
We removed the no-indexes limitation, though the "replace a row" limitation above includes unique secondary indexes. The limitation is "no imported row may conflict with an existing row (in the primary or any unique secondary index) or the import will be rejected."